src/Microsoft.ComponentDetection.Orchestrator/Orchestrator.cs

```diff
@@ -55,6 +59,15 @@ public async Task<ScanResult> LoadAsync(string[] args, CancellationToken cancell
             baseArguments = new BaseArguments();
         }
 
+        // Set the minimum logging level
+        MinimumLogLevelSwitch.MinimumLevel = baseArguments.Verbosity switch
+        {
+            VerbosityMode.Quiet => LogEventLevel.Error,
+            VerbosityMode.Normal => LogEventLevel.Information,
+            VerbosityMode.Verbose => LogEventLevel.Debug,
+            _ => throw new ArgumentOutOfRangeException(nameof(baseArguments.Verbosity), "Invalid verbosity level"),
+        };
```

Comment:
Thinking out loud here: should we line up our verbosity mode names with serilog's logging levels? In a future PR, because that is a breaking change.

Reply:
#442